weekly-mission(week2)/src/constants/dom.js

```diff
@@ -0,0 +1,8 @@
+const $ = (selector, element = document) => element.querySelector(selector);
+const emailInput = $('.email');
+const passwordInput = $('.password');
+const passwordConfirmInput = $('.password-confirm');
+const loginForm = $('.login-form');
+const eyeIcons = document.querySelectorAll('.eye-icon');
```

Comment:
DOM을 가져오는 함수가 어떤 것은 추상화($)가 되어있고 어떤 것은 `document.querySelectorAll`로 되어 있어서 헷갈릴 수 있다는 생각이 드는데요~

아래와 같은 느낌은 어떨까 싶어요!
```suggestion
const eyeIcons = $$('.eye-icon');
```

혹은 요런 느낌도 괜찮을 것 같습니다!
```suggestion
const eyeIcons = $all('.eye-icon');
```

Reply:
오 너무 좋습니다